homeassistant/components/verisure/camera.py

```diff
@@ -54,6 +54,7 @@ def __init__(
     ):
         """Initialize Verisure File Camera component."""
         super().__init__(coordinator)
+        Camera.__init__(self)
```

Comment:
We should probably add a super call in the init method of the `CoordinatorEntity` to make it compatible with multiple inheritence.

Reply:
Sounds like a good idea to me. Are you guys going to do that or do you want me to add it? I'm not sure how it would impact other integrations though.

Reply:
> It would be good to solve it more generally 

A quick glance at the codebase, this is being solved like this way more often. That is a wider cleanup.

In this case, lets fix it for now.